pkgs/README.md

```diff
@@ -347,7 +347,7 @@ There are a few naming guidelines:
 
 - The `pname` and the `version` attribute _must not_ contain uppercase letters — e.g., `"mplayer"` instead of `"MPlayer"`.
 
-- The `version` attribute _must_ start with a digit e.g., `"0.3.1rc2"`.
+- The `version` attribute _should_ start with a digit e.g., `"0.3.1rc2"` or `"0-unstable-1970-01-01"`.
```

Comment:
Missed the second part of this earlier. I'm fine with having just 9d200e24e396b1ad4d042865b21455191578f973 without the `must` -> `should` change.

Reply:
I'll open a new PR with that change cherry-picked.

Reply:
https://github.com/NixOS/nixpkgs/pull/333041 is now merged, seems like this one can be closed?

Reply:
Yea, it seems like feedback for this PR has been mostly negative.